### PR TITLE
Add export functions

### PR DESCRIPTION
**Pull request by nisemaru**
Hi coavins. 
I am sorry to be late.
Previously, I sent a message on nexusmods.
Please check and merge the Export feature!!

My WebSite is made with this export function.
https://bonjin-hunter-db.com/

### CODE REVIEW

**Review by coavins — CHANGES_REQUESTED** [review]
There are some small changes that I think should be made, but otherwise this looks good. Let me know what you think about my suggestions, and I will merge when you're ready.

File: src/autorun/mhrise-coavins-dps.lua [hunk 1]
```diff
@@ -150,12 +150,19 @@ local function dpsFrame()
...
 		CORE.changeOverlayVisibility('SHOW_OVERLAY_IN_QUEST')
+		CORE.getQuestNo()
+		CORE.getQuestMainMonsterId()
 
 	-- Entered post-quest sequence
 	elseif STATE.IS_POST_QUEST and not wasPostQuest then
 		CORE.log_info('quest complete')
 
 		CORE.changeOverlayVisibility('SHOW_OVERLAY_POST_QUEST')
+		CORE.getWeaponInfo()
```
[the remark this answers is not in the document]
Comment (nisemaru) [reply src/autorun/mhrise-coavins-dps.lua:161]: I agree. I have confirmed that there is no problem with the above operation.

File: src/autorun/mhrise-coavins-dps/export.lua [hunk 2]
```diff
@@ -1,55 +1,155 @@
 local STATE = require 'mhrise-coavins-dps.state'
 local CORE  = require 'mhrise-coavins-dps.core'
+local ENUM  = require 'mhrise-coavins-dps.enum'
 
 local this = {}
 
-this.exportData = function()
-	local file = {}
+this.round = function(x, n)
```
[the remark this answers is not in the document]
Comment (nisemaru) [reply src/autorun/mhrise-coavins-dps/export.lua:7]: I forgot to delete it, so please remove `n`.

**Review by coavins — APPROVED** [review]
Looks good, sorry for the delay on this. I will go ahead with the merge now.

**Comment by coavins** [comment]
I have posted version 2.12 to Nexus mods which includes these changes.

**Pull request merged**